src/main/java/com/example/controller/SimulationController.java

```diff
@@ -98,6 +102,7 @@ public class SimulationController {
     public void show() {
         parent.setVisible(true);
         parent.setManaged(true);
+        initWarning();
```

Comment:
Warning is displayed in a place that was reserved for algorithm information. We will have to manage its layout somehow. Please check https://github.com/Camel3333/Inzynierka_ByzantineGenerals/pull/63 . This PR adds algorithm information, maybe it will help to find a suitable solution.